### PR TITLE
✨ Create Input Fields

### LINKED ISSUE #14: username input field
Input field to enter your username

### LINKED ISSUE #15: Password input field
Input field to enter your Password for Login

### PR DESCRIPTION
**Pull request by mrana17**
- Username Input
- Password Input
- Add to storybook
- Styled all components

close #14
close #15

### CODE REVIEW

File: components/inputField/input.module.css [hunk 1]
```diff
@@ -0,0 +1,13 @@
+.input {
+  border-radius: 2rem;
+  border: 3px solid #b7cfcf;
```
Comment (RonjaMeinecke) [inline components/inputField/input.module.css]:
```suggestion
  border: 0.188rem solid #b7cfcf;
```

File: components/inputField/input.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+export const Username = Template.bind({});
+Username.args = {
+  placeholder: "Benutzername",
+  type: "text",
+};
+
+export const Password = Template.bind({});
+Password.args = {
+  placeholder: "Password",
```
Comment (RonjaMeinecke) [inline components/inputField/input.stories.tsx]:
```suggestion
  placeholder: "Passwort",
```

to be consistent with your other placeholders that are in german

**Review by RonjaMeinecke — APPROVED** [review]
👌🏼

**Review by sweisz — COMMENTED** [review]
LGTM 💯

**Pull request merged**